### PR TITLE
Section content update

### PR DESCRIPTION
**Pull request by adborden**

### CODE REVIEW

**Comment by adborden** [comment]
Ready for review.

File: js/util/request_form/form_sections.js [hunk 1]
```diff
@@ -54,8 +54,8 @@ const FORM_SECTIONS = [
     title: 'Your request',
     description: domify(`
       The description of the records you are requesting is important. The scope
-      of your request can impact how quickly an agency could respond to your
-      request. Your description should be as clear and specific as possible
+      of your request can impact how quickly an agency can respond to your
+      request. Your description should be as  clear and specific as possible
```
Comment (hbillings) [inline js/util/request_form/form_sections.js:58]: Extra space before "clear."

File: js/util/request_form/form_sections.js [hunk 2]
```diff
@@ -90,15 +91,20 @@ const FORM_SECTIONS = [
...
       it is likely to contribute significantly to public understanding of the
-      operations and activities of the government, and is not primarily in the
-      commercial interest of the requester.
+      operations and activities of the government and is not primarily in the
+      commercial interest of the requester.  Requests for fee waivers from
+      individuals who are seeking records pertaining to themselves usually do
+      not meet this standard.  Additionally, a requester’s inability to pay
+      fees is not a legal basis for granting a fee waiver.  When making a
+      request for a fee waiver you must explain how your request meets the
+      standard described above.
```
Comment (hbillings) [inline js/util/request_form/form_sections.js:107]: Double spacing between all of these sentences.

File: js/util/request_form/form_sections.js [hunk 3]
```diff
@@ -115,18 +121,26 @@ const FORM_SECTIONS = [
...
+      establishing that the  failure to obtain the records on an expedited
+      basis “could reasonably be expected to pose an imminent threat to the
+      life or physical safety of an individual;” or, (2) if the requester is a
+      “person primarily engaged in  disseminating information,” by
+      demonstrating that there exists an “urgency to inform the public
+      concerning actual or alleged Federal Government activity.”  Agencies may
+      also establish additional criteria for granting expedited processing,
+      which they describe in their FOIA regulations.  Requesters must
+      demonstrate how they meet one of the standards in order to have their
+      request processed on an expedited basis.
```
Comment (hbillings) [inline js/util/request_form/form_sections.js:143]: Lots of double spaces in here too!

**Review by hbillings — APPROVED** [review]

**Comment by adborden** [comment]
I think all the double-spaces before sentences are okay since this is going into html and it will compress the whitespace.

**Pull request merged**